video_xblock/backends/html5.py

```diff
@@ -56,6 +56,8 @@ def get_frag(self, **context):
         """
         data_setup = Html5Player.player_data_setup(context)
         data_setup['sources'][0]['type'] = self.get_type(context['url'])
+        data_setup['html5'] = {'nativeTextTracks': False}
```

Comment:
@ihor-romaniuk Could we have any tests related to that change?

Reply:
If we need to add some backend test, it's better to delegate this task to him.

Reply:
I think we should pass the task to the backend dev to cover Python changes but I don't know yet what to cover

Reply:
Will reassign the issue to @dyudyunov please clarify how we could test that code.

Reply:
Note for history:

Video xblock's tests are broken and should be fixed globally. We've decided to go without tests to not block the fix.